### DISCUSSION TITLE
Problem connecting to the wallet

### DISCUSSION THREAD

**Discussion by lamadogs** [WalletConnect/walletconnect-monorepo#4256] [Developer Support]
Good morning, yesterday i was trying connecting my wallet to mergeinterswap website (xxx) to solve an issue. But when i try to connect my wallet using WalletConnect it writes that there was an error connecting to the service. Someone knows how to possibly solve the issue ?

**Comment by glitch-txs** [comment]
That's a scam website, do not interact with it please.
